api/api.go

```diff
@@ -2,7 +2,11 @@ package api
 
 import (
 	"fmt"
+	"net/http"
 	"reflect"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 
```

Comment:
remove this empty line and run goimports again

Reply:
done